metaflow/plugins/argo/argo_inner_flow.py

```diff
@@ -1,31 +1,64 @@
 from metaflow.plugins.argo.argo_client import ArgoClient
 from metaflow.metaflow_config import KUBERNETES_NAMESPACE
+import time
+
+### ADDING WAIT FEATURE AND UPDATING PROPERTIES
 
 
 class TriggeredRun:
+    """
+    This class takes in information to identify and trigger a run of a
```

Comment:
To help us best review this PR and ensure the class's usability, could you also include a unit test example use of this class?

For example, you can add a tutorial to https://github.com/zillow/metaflow/blob/b6afc695eda124c21fcbfd73b78e5a0b5eb05271/metaflow/tutorials